server/testSubmission2.json

```diff
@@ -0,0 +1,15 @@
+{
+  "state": "submitted",
+  "attributes": {
+    "title": "A Test Hack 2",
+    "description": "Our hack is very very cool! I like this hack, and my teammates like it too.",
+    "technologies": ["MongoDB", "JSON"]
+  },
+  "urls": [
+    { "label": "GitHub Repository", "url": "https://github.com/hackru/sledge" },
+    { "label": "YouTube Video Demo", "url": "https://youtube.com/idk" }
+  ],
+  "categories": [
+    { "categoryID": "1236abcdef", "categoryName": "Best Beginner Hack" }
+  ]
+}
```

Comment:
Hm okay, I'm gonna need you to make a decision on either creating an entirely new file for modifying the testSubmission, or just modifying it as a variable inside the test file